lib/Request.php

```diff
@@ -275,18 +275,18 @@ function setRawServerData(array $data) {
      */
     function __toString() : string {
 
-        $out = $this->getMethod() . ' ' . $this->getUrl() . ' HTTP/' . $this->getHTTPVersion() . "\r\n";
-
+        $eol = PHP_EOL;
```

Comment:
Per spec all http headers need to be separated by \r\n.

This is not subject to cross plattform problems